lib/utility.js

```diff
@@ -1,10 +1,171 @@
 import Product from "../models/Product";
 
-// NOTE: variables and functions whose name starts with "_" are private and not to be exported
+// const marketUrl = "http://testcusa-env.afejb3cmmd.us-east-2.elasticbeanstalk.com/MarketExecute.php";
+// const loginUrl = "http://testcusa-env.afejb3cmmd.us-east-2.elasticbeanstalk.com/LoginExecute.php"
+const marketUrl = "https://madishare.com/MarketExecute.php";
+const loginUrl = "https://madishare.com/LoginExecute.php";
+let app = getApp();
 
-const _marketUrl = "http://testcusa-env.afejb3cmmd.us-east-2.elasticbeanstalk.com/MarketExecute.php";
+/** @review
+ * Local function to promisify wx request 

```

Comment:
Can we mention it does file upload when "filePath" is provided?
When I first read it, I got really confused why filePath is there.
It was only after reading the code that I realize this function also does uploadFile()